search/views.py

```diff
@@ -36,6 +37,17 @@ class SchoolViewSet(FilterWithBooleanAndSearchMixin, viewsets.ReadOnlyModelViewS
         if f.name not in ["specialised_divisions", "data", "school_name"]
     ]
 
+class SchoolLocationViewSet(FilterWithBooleanAndSearchMixin, viewsets.ReadOnlyModelViewSet):
+    queryset = School.objects.all()
+    serializers = SchoolSerializer
+    serializer_class = SchoolLocationSerializer
+    pagination_class = None
+    filterset_fields = [
+        f.name
+        for f in School._meta.fields
```

Comment:
`_` means that a field is "private", Django internals in this case, should not be used